### ISSUE TITLE
Screenreader users

### ISSUE THREAD

**Issue by ghost** [fac19/week3-JICG#3]
- [x] can it be used with only a screenreader?
- [x] screenreaders should be able to clearly distinguish between todo content, todo buttons, the input field and the submit button
- [x] when using a screen reader, it should be clear both what a button does and what todo it is related to.
- [x] screenreaders should be able to match up todos with the correct buttons
- [x] contrast colors
